### PR TITLE
Some changes in notes for message and main file

### PR DESCRIPTION
**Pull request by shivanshu1086**

### CODE REVIEW

**Comment by Poolitzer** [comment]
Hey, thanks for the PR. I fail to see the issue though. The and makes no sense there (or I am stupid), and the HTML markdown we talk about in the other file is not HTML5. Its just basic html entities. Nothing wrong with letting it stay HTML.

File: telegram/chat.py [hunk 1]
```diff
@@ -208,8 +208,8 @@ def kick_member(self, *args, **kwargs):
             :obj:`bool`: If the action was sent succesfully.
 
         Note:
-            This method will only work if the `All Members Are Admins` setting is off in the
-            target group. Otherwise members may only be removed by the group's creator or by the
+            This method will only work, if the `All The Members Are Admins` and setting is off in the
```
Comment (JosXa) [inline telegram/chat.py]: Please revert this line

File: telegram/chat.py [hunk 2]
```diff
@@ -208,8 +208,8 @@ def kick_member(self, *args, **kwargs):
...
+            target group. Otherwise, members may only be removed by the group's creator or by the
```
Comment (JosXa) [inline telegram/chat.py]: ok yeah, nice

**Comment by tsnoam** [comment]
Hi,

Thank you for your sincere desire to improve the project documentation.
We take every contribution with the highest respect.

However, after discussing this PR in the dev group we've decided that the current documentation is correct both semantically (HTML vs HTML5) and with its grammar. Therefor we're afraid we'll have to reject this PR.

Again, thank you for your contribution.

**Pull request closed without merging**